presto-release-tools/src/test/java/com/facebook/presto/release/tasks/TestFinalizeReleaseTask.java

```diff
@@ -41,11 +48,26 @@
     private static class MockGit
             extends NoOpGit
     {
+        private Optional<Consumer<Optional<String>>> checkoutAction = Optional.empty();
```

Comment:
Isn't the convention more to have inner classes at the bottom so that the fields of the enclosing class come first?

Reply:
I don't think there is a hard guideline about that. I took a look at the Presto code base and see most static inner class either at the bottom, or right after the declaration of the fields.

Anyway, I'm moving the static inner class to a standalone class for sharing across different tests.